handbook/engineering/deployments/playbooks.md

```diff
@@ -0,0 +1,68 @@
+
+# Rolling back sourcegraph.com
+
+To roll back soucegraph.com, push a new commit to the `release` branch in [deploy-sourcegraph-dot-com](https://github.com/sourcegraph/deploy-sourcegraph-dot-com) that reverts the image tags and configuration to the desired state.
+
+```sh
+# Ensure that you're up-to-date
+git checkout release
+git pull
+
+# Rollback the release branch to $COMMIT
+# See https://stackoverflow.com/a/21718540 if you want more context
+git revert --no-commit $COMMIT..HEAD
+
+# Push the revert commit back to the release branch
+git commit
+git push origin release
+```
+
+[Buildkite](https://buildkite.com/sourcegraph/deploy-sourcegraph-dot-com/) will deploy the working commit to sourcegraph.com.
+
+🚨 You also need to disable auto-deploys to prevent Renovate from automatically merging in image digest updates so that the site doesn't roll-forward.
+
+1. Go to [renovate.json](https://github.com/sourcegraph/deploy-sourcegraph-dot-com/blob/release/renovate.json) and remove the `"extends:["default:automergeDigest"]` entry for the "Sourcegraph Docker images" group ([example](https://github.com/sourcegraph/deploy-sourcegraph-dot-com/commit/0eb16fd9e3ddfcf3a3c75ccdda0e7eddabf19c7a)).
+1. Once you have fixed the issue in the `main` branch of [sourcegraph/sourcegraph](https://github.com/sourcegraph/sourcegraph), re-enable auto-deploys by reverting your change to [renovate.json](https://github.com/sourcegraph/deploy-sourcegraph-dot-com/blob/release/renovate.json) from step 1.
+
+# Accessing sourcegraph.com database
+
+## Via the CLI
+
+Sourcegraph.com utilizes an external HA database. You will need to connect to it directly. The easiest way to do this is through the `gcloud` cli.
+
+To connect to the production database:
+
+```
+  gcloud beta sql connect sg-cloud-732a936743 --user=sg -d sg --project sourcegraph-dev
+```
+
+However, if you want to use any other SQL client, you'll have to run the [`cloud_sql_proxy`](https://cloud.google.com/sql/docs/postgres/connect-admin-proxy#install) utility, which authenticates with you local `gcloud` credentials automatically.
+
+```
+  cloud_sql_proxy -instances=sourcegraph-dev:us-central1:sg-cloud-732a936743=tcp:5555
+```
+
+Once the proxy connects successfully, you can use any client to connect to the local `5555` port (you can choose any other port you want).
+
+The password of the sg user is in our shared 1Password under [Google Cloud SQL](https://team-sourcegraph.1password.com/vaults/dnrhbauihkhjs5ag6vszsme45a/allitems/svfiw4vcbxhhbobpl442olyebu)
+
+## Via BigQuery (for read-only operations)
+
+You can also query the production database via BigQuery as an external data source.
+
+See an [example query](https://console.cloud.google.com/bigquery?sq=527047051561:bfa7c7e57f884d209f261d15e4610229) to get started.
+
+**Note**: This method only permits read-only access
+
+# Manage users in k8s.sgdev.org
```

Comment:
Playbooks like this (and the dot-com one) imo should just go in the main deployments page, under each deployment where they used to be, to help with discoverability and avoid confusion - I think anything on separate pages like this one should not be deployment-specific

(a bit late on the review, but happy to put in a PR if you don't disagree)

Reply:
I disagree, the document was quite hard to GROK as it had a mix of descriptions of the deployments, steps to manage things, etc. Maybe we could group them per-deployment in the playbook section if they are for a particular deployment, which would potentially fix both our concerns, wdyt?

I would agree to generalize those steps, as most of them should be fairly generic for the most part, but I chose in this PR to keep the same content as I was already moving a lot of things around which make the PR harder to review.

Reply:
In that case per-deployment playbook sections + links to those sections from the main deployments page would be a good middle ground 👍

Reply:
Maybe some further context, when I moved them I also left the names there as it would make it easier to use search for `k8s.sgdev.org` and the playbook doc would pop up, helping discoverability.